vision/src/autogluon/vision/configs/presets_configs.py

```diff
@@ -64,7 +64,7 @@ def _call(*args, **kwargs):
             'early_stop_patience': 5
             },
         'hyperparameter_tune_kwargs': {
-            'num_trials': 8,
+            'num_trials': 1,
```

Comment:
Will this crash? How is this different from deleting the entire `'hyperparameter_tune_kwargs'` key?

Reply:
no, if deleting the hypertune kwargs it's running without fork server where errors will be thrown directly to main thread, not quite sure which one is better, but I am gonna delete it to avoid furthur issues